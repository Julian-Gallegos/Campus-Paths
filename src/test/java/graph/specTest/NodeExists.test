# Tests that nodeExists properly returns true if the node exists

# Create an empty graph
CreateEmptyGraph graph1

# Add a node
AddNode graph1 node1

# See if node1 exists
NodeExists graph1 node1