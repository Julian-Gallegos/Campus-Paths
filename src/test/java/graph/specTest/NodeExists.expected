# Tests that nodeExists properly returns true if the node exists

#Create an empty graph
created graph graph1

# Add a node
added node node1 to graph1

# See if node1 exists
node1 in graph1 exists(true)